### PR TITLE
Add new "MarkAs" methods and unit tests for it.

### PR DESCRIPTION
**Pull request by MatusTomko**

### CODE REVIEW

File: src/Extensions/Abstractions/Activities/ActivityExtensions.cs [hunk 1]
```diff
@@ -60,6 +60,24 @@ public static class ActivityExtensions
 		public static Activity SetResult(this Activity activity, TimedScopeResult result) =>
 			activity.SetTag(ActivityTagKeys.Result, ActivityResultStrings.ResultToString(result));
 
+		/// <summary>
+		/// MarkAsSuccess
```
Comment (artempushkin) [inline src/Extensions/Abstractions/Activities/ActivityExtensions.cs:64]: Please make more human readable, I suggest: Set activity result to Success. Similar for the other 2 methods.

File: tests/Extensions/Abstractions.UnitTests/Activities/ActivityExtensionsTests.cs [hunk 2]
```diff
@@ -116,6 +116,42 @@ public void SetResult_SetsValue(TimedScopeResult result)
 			Assert.IsNull(activity2.GetTag(ActivityTagKeys.Result));
 		}
 
+		[TestMethod]
+		public void MarkAsSuccess_SetSuccess()
```
Comment (artempushkin) [inline tests/Extensions/Abstractions.UnitTests/Activities/ActivityExtensionsTests.cs:120]: Please change to Set**s**ResultToSuccess. Similar for the other 2 tests.

**Review by artempushkin — CHANGES_REQUESTED** [review]

**Review by artempushkin — APPROVED** [review]
Looks great!

**Review by AndreyTretyak — APPROVED** [review]

**Comment by AndreyTretyak** [comment]
@MatusTomko  Thank you for your contribution!

**Pull request merged**